_es/tour/multiple-parameter-lists.md

```diff
@@ -29,10 +52,78 @@ Aquí se muestra un ejemplo:
       println(filter(nums, modN(2)))
       println(filter(nums, modN(3)))
     }
+```
+{% endscalafiddle %}
 
-_Nota: el método `modN` está parcialmente aplicado en las dos llamadas a `filter`; esto significa que solo su primer argumento es realmente aplicado. El término `modN(2)` devuelve una función de tipo `Int => Boolean` y es por eso un posible candidato para el segundo argumento de la función `filter`_
+_Nota: el método `modN` está parcialmente aplicado en las dos llamadas a `filter`; esto significa que solo su primer argumento es realmente aplicado. El término `modN(2)` devuelve una función de tipo `Int => Boolean` y es por eso un posible candidato para el segundo argumento de la función `filter`._
 
 Aquí se muestra la salida del programa anterior:
 
     List(2,4,6,8)
     List(3,6)
+
+
+### Casos de uso
+
+Casos de uso sugeridos para múltiples listas de parámetros incluyen:
+
+#### Inferencia de tipos
+
+En Scala, la inferencia de tipos se realiza parámetro a parámetro.
+Suponer que se dispone del siguiente método:
+
+```tut
+def foldLeft1[A, B](as: List[A], b0: B, op: (B, A) => B) = ???
+```
+
+Y se desea invocarlo de la siguiente manera, pero puede comprobar que no compila:
+
+```tut:fail
+def notPossible = foldLeft1(numbers, 0, _ + _)
+```
+
+Debes invocarlo de alguna de las maneras propuestas a continuación:
+
+```tut
+def firstWay = foldLeft1[Int, Int](numbers, 0, _ + _)
+def secondWay = foldLeft1(numbers, 0, (a: Int, b: Int) => a + b)
+```
+
+Esto se debe a que Scala no será capaz de inferir el tipo de la función `_ + _`, como está aún infiriendo `A` y `B`.
+Moviéndo el parámetro `op` a su propia lista de parámetros, los tipos de `A` y `B` son inferidos en la primera lista de parámetros.
```

Comment:
Mispring: The term "Moviéndo" in spanish does not have accent mark (it has to be Moviendo).